pkg/apps/apps.go

```diff
@@ -43,7 +43,7 @@ const (
 const (
 	OnboardingSlug = "onboarding"
 	HomeSlug       = "home"
-	FilesSlug       = "files"
+	FilesSlug      = "files"
```

Comment:
The golang developers have gofmt called when they save a file, and it does this stuff for us. It is really cool in golang to not have to think about code style nitpicks like how many spaces for indentation, and just let gofmt does this work for us!

To be honest, we don't use gofmt, but goimports. It is the same, but it also adds the imports automatically.